<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>851</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Babel</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>791</width>
      <height>871</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="main_page">
     <widget class="QPushButton" name="login">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>380</y>
        <width>241</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">		margin-top: 15px;
		margin-bottom: 15px;
		background: #0c56a9;
		border: 1px solid #08298A;
		border-radius: 5px;
		font-size : 15px;
	color: #FFFFFF;
	font-family: grobold;</string>
      </property>
      <property name="text">
       <string>Log In</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>276</y>
        <width>55</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#FFFFFF;
font-size: 15 px;
font-family:Segoe Ui;</string>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="checkpass">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>220</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #777;
		background: rgba(180,180,180,.9);
	border: 1px solid #08298A;
		height: 2.6em;
		padding: 10px 40px;
		border-radius: 5px;
</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>210</y>
        <width>55</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>200</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #FFFFFF;
font-size: 15;

font-family:Segoe Ui;</string>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="chekUser">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>300</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #777;
		background: rgba(180,180,180,.9);
			border: 1px solid #08298A;
		height: 2.6em;
		padding: 10px 40px;
		border-radius: 5px;
</string>
      </property>
     </widget>
     <widget class="QLabel" name="name_babel">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>90</y>
        <width>241</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 50px;
color : white;
font-family : ./Font/GROBOLD.ttf</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_back">
      <property name="geometry">
       <rect>
        <x>-100</x>
        <y>-140</y>
        <width>771</width>
        <height>821</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>530</y>
        <width>93</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">		margin-top: 15px;
		margin-bottom: 15px;
		background: #0c56a9;
		border: 1px solid #08298A;
		border-radius: 5px;
		font-size : 15px;
	color: #FFFFFF;
font-family:grobold;</string>
      </property>
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
     <zorder>label_back</zorder>
     <zorder>label_3</zorder>
     <zorder>label_4</zorder>
     <zorder>label_2</zorder>
     <zorder>name_babel</zorder>
     <zorder>login</zorder>
     <zorder>checkpass</zorder>
     <zorder>chekUser</zorder>
     <zorder>pushButton_3</zorder>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_ground">
      <property name="geometry">
       <rect>
        <x>-70</x>
        <y>-100</y>
        <width>891</width>
        <height>751</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>130</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-family: grobold;
font-size:50px;
color: white;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="name_wait">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>230</y>
        <width>261</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-family : Segoe Ui;
font-size : 30px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>270</y>
        <width>71</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color : white;
font-family : Segoe Ui;
font-size: 50px;</string>
      </property>
      <property name="text">
       <string>. . .</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>400</y>
        <width>141</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">		margin-top: 15px;
		margin-bottom: 15px;
		background: #0c56a9;
		border: 1px solid #08298A;
		border-radius: 5px;
		font-size : 15px;
	color: #FFFFFF;
	font-family: Segoe Ui;</string>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>-350</x>
        <y>10</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:30px;
font-family:grobold;
color: white;
</string>
      </property>
      <property name="text">
       <string>BABEL</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>-180</x>
        <y>540</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../call.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_back2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-20</y>
        <width>671</width>
        <height>671</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>181</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:50px;
font-family:grobold;
color:white;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="discobutton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>90</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">		margin-top: 15px;
		margin-bottom: 15px;
		background: #0c56a9;
		border: 1px solid #08298A;
		border-radius: 5px;
		font-size : 15px;
	color: #FFFFFF;
font-family:Segoe UI
</string>
      </property>
      <property name="text">
       <string>Disconnect</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Options">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>40</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">		margin-top: 15px;
		margin-bottom: 15px;
		background: #0c56a9;
		border: 1px solid #08298A;
		border-radius: 5px;
		font-size : 15px;
	color: #FFFFFF;
font-family: Segoe Ui, bold;
</string>
      </property>
      <property name="text">
       <string>Options</string>
      </property>
     </widget>
     <widget class="QLabel" name="login_name">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>170</y>
        <width>291</width>
        <height>37</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:20px;
color:white;
font-family:Segoe Ui;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>130</y>
        <width>91</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../salut.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="call_button">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>460</y>
        <width>82</width>
        <height>82</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> background-color: #298A08;
 border-style: solid;
 border-width:1px;
 border-radius:40px;
 border-color: #298A08;
 max-width:80px;
 max-height:80px;
 min-width:80px;
 min-height:80px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>230</y>
        <width>381</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">		margin-top: 15px;
		margin-bottom: 15px;
		border: 1px solid #08298A;
		border-radius: 10px;
		background:#EFF8FB;
		font-size : 15px;</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>250</y>
        <width>361</width>
        <height>181</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>359</width>
         <height>179</height>
        </rect>
       </property>
      </widget>
     </widget>
     <zorder>scrollArea</zorder>
     <zorder>label_back2</zorder>
     <zorder>label_8</zorder>
     <zorder>label_6</zorder>
     <zorder>label_10</zorder>
     <zorder>discobutton</zorder>
     <zorder>Options</zorder>
     <zorder>login_name</zorder>
     <zorder>label_11</zorder>
     <zorder>call_button</zorder>
     <zorder>listWidget</zorder>
    </widget>
    <widget class="QWidget" name="demand_call">
     <widget class="QLabel" name="background">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>961</width>
        <height>871</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="name_logo">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:40px;
font-family:grobold;
color:white;</string>
      </property>
      <property name="text">
       <string>BABEL</string>
      </property>
     </widget>
     <widget class="QLabel" name="nameofuser">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:20px;
color:white;
font-family:Segoe Ui;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>130</y>
        <width>541</width>
        <height>211</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">		margin-top: 15px;
		margin-bottom: 15px;
		border: 1px solid #08298A;
		border-radius: 10px;
		background:#333333;
		font-size : 15px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="name_collegue">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>90</y>
        <width>391</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:20px;
color:white;
font-family:Segoe Ui;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="buton_options">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>20</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">		margin-top: 15px;
		margin-bottom: 15px;
		background: #0c56a9;
		border: 1px solid #08298A;
		border-radius: 5px;
		font-size : 15px;
	color: #FFFFFF;
font-family:Segoe Ui;</string>
      </property>
      <property name="text">
       <string>Options</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_disonnects">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>20</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">		margin-top: 15px;
		margin-bottom: 15px;
		background: #0c56a9;
		border: 1px solid #08298A;
		border-radius: 5px;
		font-size : 15px;
	color: #FFFFFF;
font-family:Segoe Ui;
</string>
      </property>
      <property name="text">
       <string>Disconnect</string>
      </property>
     </widget>
     <widget class="QPushButton" name="plus_buton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>590</y>
        <width>82</width>
        <height>82</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> background-color: white;
 border-style: solid;
 border-width:1px;
 border-radius:40px;
 border-color: blue;
 max-width:80px;
 max-height:80px;
 min-width:80px;
 min-height:80px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="userss">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>150</y>
        <width>181</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Images/gens.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>250</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:50px;
color:white;</string>
      </property>
      <property name="text">
       <string>02 : 14</string>
      </property>
     </widget>
     <widget class="QListView" name="display_chat">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>430</y>
        <width>531</width>
        <height>251</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">		margin-top: 15px;
		margin-bottom: 15px;
		border: 1px solid #08298A;
		border-radius: 10px;
		background:#EFF8FB;
		font-size : 15px;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="micro_button">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>340</y>
        <width>82</width>
        <height>82</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> background-color: #298A08;
 border-style: solid;
 border-width:1px;
 border-radius:40px;
 border-color: #298A08;
 max-width:80px;
 max-height:80px;
 min-width:80px;
 min-height:80px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="racroch_button">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>340</y>
        <width>82</width>
        <height>82</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> background-color: #B40404;
 border-style: solid;
 border-width:1px;
 border-radius:40px;
 border-color: #B40404;
 max-width:80px;
 max-height:80px;
 min-width:80px;
 min-height:80px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>690</y>
        <width>451</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>680</y>
        <width>81</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">		margin-top: 15px;
		margin-bottom: 15px;
		background: #0c56a9;
		border: 1px solid #08298A;
		border-radius: 5px;
		font-size : 15px;
	color: #FFFFFF;
font-family:Segoe Ui;</string>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QListView" name="listView">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>131</width>
        <height>451</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">		margin-top: 15px;
		margin-bottom: 15px;
		border: 1px solid #08298A;
		border-radius: 10px;
		background:#EFF8FB;
		font-size : 15px;</string>
      </property>
     </widget>
     <zorder>background</zorder>
     <zorder>label_5</zorder>
     <zorder>name_logo</zorder>
     <zorder>nameofuser</zorder>
     <zorder>name_collegue</zorder>
     <zorder>buton_options</zorder>
     <zorder>button_disonnects</zorder>
     <zorder>plus_buton</zorder>
     <zorder>label_12</zorder>
     <zorder>userss</zorder>
     <zorder>display_chat</zorder>
     <zorder>micro_button</zorder>
     <zorder>racroch_button</zorder>
     <zorder>lineEdit</zorder>
     <zorder>pushButton_4</zorder>
     <zorder>listView</zorder>
    </widget>
    <widget class="QWidget" name="iwantcallyou">
     <widget class="QLabel" name="label_back3">
      <property name="geometry">
       <rect>
        <x>-100</x>
        <y>-60</y>
        <width>801</width>
        <height>721</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size : 50px;
font-family: grobold;
color : white;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="name_wants">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>270</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;
font-size:20px;
font-family:Segoe Ui;
</string>
      </property>
      <property name="text">
       <string>Incomming call Julien</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>320</y>
        <width>82</width>
        <height>112</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">		margin-top: 15px;
		margin-bottom: 15px;
		background: green;
		font-size : 15px;
	color: #FFFFFF;
 border-style: solid;
 border-width:1px;
 border-radius:40px;
 max-width:80px;
 max-height:80px;
 min-width:80px;
 min-height:80px;
font-family:Segoe Ui;</string>
      </property>
      <property name="text">
       <string>Go !</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>320</y>
        <width>82</width>
        <height>112</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">		margin-top: 15px;
		margin-bottom: 15px;
		background: #B40404;
		font-size : 15px;
	color: #FFFFFF;
 border-style: solid;
 border-width:1px;
 border-radius:40px;
 max-width:80px;
 max-height:80px;
 min-width:80px;
 min-height:80px;
font-family:Segoe Ui;</string>
      </property>
      <property name="text">
       <string>No !</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>180</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-family: grobold;
font-size: 40px;
color: white;</string>
      </property>
      <property name="text">
       <string>BABEL</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
